doc/classes/Image.xml

```diff
@@ -238,51 +253,56 @@
 			<argument index="0" name="rect" type="Rect2">
 			</argument>
 			<description>
-				Return a new [code]Image[/code] that is a copy of "area" in this [code]Image[/code].
+				Returns a new image that is a copy of the image's area specified with [code]rect[/code].
 			</description>
 		</method>
 		<method name="get_size" qualifiers="const">
 			<return type="Vector2">
 			</return>
 			<description>
+				Returns a [code]Vector2[/code] that is the size of the image.
 			</description>
 		</method>
 		<method name="get_used_rect" qualifiers="const">
 			<return type="Rect2">
 			</return>
 			<description>
-				Return the area of this [code]Image[/code] that is used/visibly colored/opaque.
+				Returns a new [code]Rect2[/code] that is the size of the image that is colored and opaque.
 			</description>
 		</method>
 		<method name="get_width" qualifiers="const">
 			<return type="int">
 			</return>
 			<description>
-				Return the width of the [code]Image[/code].
+				Returns the width of the image.
 			</description>
 		</method>
 		<method name="has_mipmaps" qualifiers="const">
 			<return type="bool">
 			</return>
 			<description>
+				Returns [code]true[/code] if the image has generated mipmaps. Returns [code]false[/code] if the image has no mipmaps.
 			</description>
 		</method>
 		<method name="is_compressed" qualifiers="const">
 			<return type="bool">
 			</return>
 			<description>
+				Returns [code]true[/code] if the image is compressed. Returns [code]false[/code] if the image is not compressed.
 			</description>
 		</method>
 		<method name="is_empty" qualifiers="const">
 			<return type="bool">
 			</return>
 			<description>
+				Returns [code]true[/code] if the image has no data.
 			</description>
 		</method>
 		<method name="is_invisible" qualifiers="const">
 			<return type="bool">
 			</return>
 			<description>
+				Returns [code]true[/code] if all the image's pixels have an alpha value of 0. Returns false if any pixel has an alpha value higher than 0.
```

Comment:
- For this one, I think it's OK to list both cases, as it's more clear that any single pixel can affect the result.
- missing [code] tags around "false"